Online-Messaging-App/src/app/register/register.component.html

```diff
@@ -1,29 +1,57 @@
 <div class="container">
-
+  <mat-toolbar color="primary" class ="navbar">
+    <div>Streamline Athletes</div>
+    <div>
+      <span>Home</span>
+      <span>About</span>
+      <span>Services</span>
+    </div>
+  </mat-toolbar>
+  <div align="center" class="mat-display-2">Streamline Athletes</div>
   <div class="row">
     <div class="col-lg-4">
       <form (ngSubmit)="registerSubmit(regform)" #regform="ngForm">
```

Comment:
Seems to be an issue were the hint to what shows up on the field doesn't show up until you click it, please fix

Reply:
Done